pages/[user].tsx

```diff
@@ -33,8 +34,8 @@ export default function User(props: inferSSRProps<typeof getServerSideProps>) {
   delete query.user; // So it doesn't display in the Link (and make tests fail)
 
   const nameOrUsername = user.name || user.username || "";
-
   const [evtsToVerify, setEvtsToVerify] = useState<EvtsToVerify>({});
+  const verified = user.verified;
```

Comment:
Personally, I'd just directly use `user.verified` and skip this extra line of code